flexmeasures/api/v3_0/tests/test_sensors_api.py

```diff
@@ -174,10 +187,10 @@ def test_fetch_sensors(
 
     print("Server responded with:\n%s" % response.json)
 
-    if test_errors:
-        assert response.status_code > 400
+    if expected_status_code > 400:
```

Comment:
lines 191 and 193 are the same now< so lose the if/else

Reply:
Without this if statement, the code from lines 201 to 215 would execute, attempting to assert conditions on the response. However, since the response in this case is an error, it causes the test case to fail.

Reply:
Okay:

```
assert response.status_code == expected_status_code
if expected_status_code == 200:
    if use_pagination:
        ....
```

Reply:
ok 👍